src-vue/src/views/ModsView.vue

```diff
@@ -63,6 +65,31 @@ export default defineComponent({
 
             this.global_load_indicator = false;
             return true;
+        },
+        async deleteMod(mod: NorthstarMod) {
+            console.log(mod)
```

Comment:
Please remove this :)

Reply:
Done with 6a974ac2eb36bf11913b5ea08fec4dcfd2e7f35e